spec/activefedora/association_spec.rb

```diff
@@ -25,10 +26,12 @@ class Image < ActiveFedora::Base
     end
 
     let(:image) { Image.create }
+    let(:image2) { Image.create }
 
     before do
       image.generic_files = [generic_file2, generic_file1]
       image.save
+      image2.save
```

Comment:
Why are we saving here? It saves two times. Calling `let!(:image2)...` would be preferable.